screens/EpisodePrediction.tsx

```diff
@@ -182,9 +126,10 @@ const EpisodePrediction: FunctionComponent<EpisodePredictionProps> = ({
                   color={Colors.avocadoGreen}
                   size={25}
                   onPress={() => {
-                    predictions.splice(idx, 1);
-                    setPredictions(predictions);
-                    setHasOneValue(predictions.some((val) => val !== ""));
+                    const newPredictions = [...predictions];
+                    newPredictions.splice(idx, 1);
+                    setPredictions(newPredictions);
```

Comment:
Not a huge deal but you could just use filter instead so you dont have to worry about the mutation issue and need to make a copy like that.
ex. setPredictions(predictions.filter((element, index) => index !== idx)

Reply:
thanks, will do!